src/Http/RequestFactory.php

```diff
@@ -185,24 +187,34 @@ public function createHttpRequest()
 			}
 		}
 
+		$remoteAddr = !empty($_SERVER['REMOTE_ADDR']) ? $_SERVER['REMOTE_ADDR'] : NULL;
+		$remoteHost = !empty($_SERVER['REMOTE_HOST']) ? $_SERVER['REMOTE_HOST'] : NULL;
```

Comment:
Because I don't think that having an empty host or address makes sense. Is that wrong assumption?